manifests/process_exporter.pp

```diff
@@ -82,12 +83,18 @@
     default => undef,
   }
 
+  if $hash_watched_processes.empty() {
```

Comment:
how is this new parameter used?

Reply:
I'll add some documentation for it but basically it's used like any other hash variable with the only requirement that the first key should be `process_names`